lib/api.go

```diff
@@ -108,36 +108,36 @@ const (
 	// remote client endpoints
 
 	// AEPush facilitates dataset push requests to a remote
-	AEPush = APIEndpoint("/push/{path:.*}")
+	AEPush = APIEndpoint("/push")
 	// AEPull facilittates dataset pull requests from a remote
 	AEPull = APIEndpoint("/pull")
 	// AEFeeds fetches and index of named feeds
 	AEFeeds = APIEndpoint("/feeds")
 	// AEPreview fetches a dataset preview from the registry
-	AEPreview = APIEndpoint("/preview/{path:.*}")
+	AEPreview = APIEndpoint("/preview")
 
 	// fsi endpoints
 
 	// AEStatus returns the filesystem dataset status
 	AEStatus = APIEndpoint("/status")
 	// AEWhatChanged returns what changed for a specific commit
-	AEWhatChanged = APIEndpoint("/whatchanged/{path:.*}")
+	AEWhatChanged = APIEndpoint("/whatchanged")
```

Comment:
between the AttributesSet changes & the way we now register routes with ref handlers, we no longer need this path suffix.